cmd/secret/cmd/root.go

```diff
@@ -181,17 +186,56 @@ func (r *root) serverCmd() *cobra.Command {
 				defer cancel()
 				err = srv.Shutdown(ctx)
 				if err != nil {
-					fmt.Println("Server Shutdown Failed", err)
+					fmt.Println("server Shutdown Failed", err)
 				}
 			}(context.Background())
 			<-shutdownCh
-			fmt.Println("Server exit")
+			fmt.Println("server exit")
 
 			return nil
 		},
 	}
 	serverCmd.Flags().StringVarP(&path, "path", "p", "file.txt", "the place where the key/value will be stored/got")
 	serverCmd.Flags().StringVarP(&port, "port", "t", "8888", "localhost address")
-
+	serverCmd.AddCommand(r.serverPingCmd())
 	return serverCmd
 }
+
+func (r *root) serverPingCmd() *cobra.Command {
+	var url string
+	var port string
+	var route string
+	var timeout time.Duration
+	var serverPingCmd = &cobra.Command{
+		Use:   "ping",
+		Short: "Check a health check route endpoint",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client := http.Client{
+				Timeout: timeout,
+			}
+			resp, err := client.Get(fmt.Sprintf("%s:%s%s", url, port, route))
+			if err != nil {
+				return fmt.Errorf("server response error: %w", err)
+			}
+			defer func() {
+				if err := resp.Body.Close(); err != nil {
+					fmt.Println("server: cannot close request body: ", err.Error())
+				}
+			}()
+
+			if resp.StatusCode != http.StatusOK {
+				responseBody, err := ioutil.ReadAll(resp.Body)
+				if err != nil {
+					return fmt.Errorf("secret client: can't get response body %w", err)
+				}
+				return fmt.Errorf("server response is not expected: body %q, wrong status code %d", responseBody, resp.StatusCode)
+			}
+			return nil
+		},
+	}
+	serverPingCmd.Flags().StringVarP(&port, "port", "p", "8880", "remote or local url port")
+	serverPingCmd.Flags().StringVarP(&route, "route", "r", "/ping", "remote or local url route")
+	serverPingCmd.Flags().StringVarP(&url, "url", "u", "http://localhost", "remote or local url for server checking")
```

Comment:
For documentation use more definite sentences. For example, `remote or local url` define all urls that exist so it meaningless to describe it that way (better to be `port to connect. Default: '8880'`).